docs/development.md

```diff
@@ -34,7 +34,7 @@ Lastly, run Ollama:
 go run . serve
 ```
 
-## Windows
+## Windows (x64)
```

Comment:
It may be simpler to keep this `Windows` but to mention in a warning below acceleration libraries aren't currently supported for ARM (vs two sections)

Reply:
Have we found an easier way to get a gcc compatible compiler wrapper on Windows ARM?  Msys2 isn't the easiest, so if we haven't found an easier one, I think we should have a bit more guidance to help users get a working setup installed.

Reply:
The top of the doc (perhaps a bit too easy to glance over) has a link to TDM-gcc which works pretty well.

Reply:
(and llvm-mingw for ARM)